pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -667,21 +666,23 @@ protected static Expression invokeCompileTimeFunctionExpression(Expression funcE
       function.getOperands().set(i, operand);
     }
     String funcName = function.getOperator();
-    if (FunctionRegistry.containsFunctionByName(funcName) && compilable) {
```

Comment:
why are you removing this check?

Reply:
`FunctionRegistry.getFunctionByName()` can handle function name not registered by returning `null`. I have a `null` check after this to save the extra hash lookup.